resources/views/page.blade.php

```diff
@@ -10,6 +10,9 @@
 @section('classes_body', $layoutHelper->makeBodyClasses())
 
 @section('layout')
+    @if(config('tablar','display_alert'))
```

Comment:
It should have resolved 2 conditions. 

1. when display_alert = true
2. when livewire = true

It's all about livewire, right?

Reply:
Hi @takielias,

Thanks for your feedback! The changes I made are not just about Livewire. I’ve also modified the normal (non-Livewire) usage of alerts to ensure they are always included as an overlay, making them more dynamic and consistent across all use cases.

This means:

When display_alert = true, the alert container will always be rendered, regardless of whether Livewire is being used or not.
For normal (non-Livewire) alerts, they now behave dynamically with improved configuration options (e.g., duration, dismissibility).
For Livewire, the alerts integrate seamlessly using JavaScript events.

If you'd prefer, I can create a separate, more focused PR that resolves the Livewire compatibility issue without modifying the normal alert behavior.

Let me know if this approach works better for you, and I’ll be happy to make the adjustments.

Reply:
@ser-tec  Thanks for the detailed explanation! For better maintainability and extensibility, would it make sense to keep the concerns separated? I think having a dedicated PR for the Livewire integration would help keep the codebase cleaner. Let me know what you think.